modules/virt-disable-individual-bootsource-update.adoc

```diff
@@ -5,13 +5,18 @@
 
 :_content-type: PROCEDURE
 [id="virt-disable-individual-bootsource-update_{context}"]
-= Disabling an automatic update for a custom boot source
+= Disabling an automatic update for a single system-defined or user-defined boot source
 
-You can reduce the number of logs on disconnected environments or reduce resource usage by disabling the automatic imports and updates for an individual custom boot source.
+You can disable automatic imports and updates for a single user-defined boot source and for a single system-defined boot source.
+
+Because system-defined boot sources are not listed by default in the `spec.dataImportCronTemplates` of the `HyperConverged` custom resource (CR), you must add the boot source and disable auto imports and updates.
 
 .Procedure
 
-* Edit the `HyperConverged` custom resource (CR) and disable updates for a custom boot source by setting the `dataimportcrontemplate.kubevirt.io/enable` annotation in the CR to `false`.
+* To disable automatic imports and updates for a single user-defined boot source, remove the boot source from the `spec.dataImportCronTemplates` field in the custom resource list.
+* To disable automatic imports and updates for a single system-defined boot source:
+** Edit the `HyperConverged` CR and add the boot source to `spec.dataImportCronTemplates`.
+** Disable automatic imports and updates by setting the `spec.dataImportCronTemplates.dataimportcrontemplate.kubevirt.io/enable` annotation to `false`. For example:
```

Comment:
`spec.dataImportCronTemplates.dataimportcrontemplate.kubevirt.io/enable` this is not correct though. Please check my last comment on this.

Reply:
Fixed